### PR TITLE
Wire paging queries into UI

### PR DESCRIPTION
**Pull request by ThePrez**

### CODE REVIEW

**Review by worksofliam — CHANGES_REQUESTED** [review]
@ThePrez Check out my comment about query history, then I will follow up with some testing

File: src/views/results/index.ts [hunk 1]
```diff
@@ -234,8 +221,8 @@ export function initialise(context: vscode.ExtensionContext) {
                   }
                 }
               }
-              
-              if (statement.type === `statement` && statement.history !== false) {
+
+              if (statement.type === `statement`) {
```
[the remark this answers is not in the document]
Comment (ThePrez) [reply src/views/results/index.ts:225]: Why should some statements not make it to the query history?

[the remark this answers is not in the document]
Comment (worksofliam) [reply src/views/results/index.ts:225]: We use this query API for things like getting advised indexes and such from the Schema Browser.

[the remark this answers is not in the document]
Comment (ThePrez) [reply src/views/results/index.ts:225]: ahh, makes sense. Thank you

[the remark this answers is not in the document]
Comment (ThePrez) [reply src/views/results/index.ts:225]: put back, https://github.com/halcyon-tech/vscode-db2i/pull/31/commits/c565870213edd074c917c1b3b9c49bdfd257e504

**Review by worksofliam — CHANGES_REQUESTED** [review]
Code looks good, but let's do some cleanup too. In `views/results/html.ts`, we can now remove these two methods as they are no longer used anywhere.

* `generateResults`
* `generateTable`

**Comment by ThePrez** [comment]
> Code looks good, but let's do some cleanup too. In `views/results/html.ts`, we can now remove these two methods as they are no longer used anywhere.
> 
>     * `generateResults`
> 
>     * `generateTable`

Good catch! https://github.com/halcyon-tech/vscode-db2i/pull/31/commits/0b245c7a10064a26fd5994ecfcd9ee7514239208

**Comment by worksofliam** [comment]
Thank you @ThePrez - beautiful PR! More deletions than additions. I wish I could give you an award

**Comment by worksofliam** [comment]
Testing complete. Great job

**Pull request merged**